### ISSUE TITLE
Website previews for NSFW posts aren't blurred

### ISSUE THREAD

**Issue by bdonvr** [aeharding/wefwef#112]
![IMG_1159](https://github.com/aeharding/wefwef/assets/10774527/4e3ea08f-aecb-4233-aa7f-c38485c106b4)

Top post is an image file and is blurred, bottom post is a preview of a website.

Here's the bottom of that post with details:

![IMG_1160](https://github.com/aeharding/wefwef/assets/10774527/429faf9c-1666-4888-8051-0f8464773e7c)

**Comment by JayK1524** [comment]
I would like to work on this issue. Please guide me, how can I resolve this.

**Comment by AmitT98** [comment]
I would also like to help in the issue if @bdonvr @JayK1524, have no objection with it.

**Comment by xpertbot** [comment]
I am an experienced full-stack Developer. I am currently looking into this issue. I already have everything spun up on my local environment and looking into how to address it best. However, if someone else would like to take a stab at it, let me know in this thread, and I can help that person instead.

**Comment by JayK1524** [comment]
> I am an experienced full-stack Developer. I am currently looking into this issue. I already have everything spun up on my local environment and looking into how to address it best. However, if someone else would like to take a stab at it, let me know in this thread, and I can help that person instead.

I wanted to resolve this issue, but you already know how to resolve it, so go for it.

**Comment by l0wigh** [comment]
I looked a bit the source code and tried some dirty fix. I hope it helps contributors with more time (and react knowledge) to fix this.

I did put blur by default to ImageContainer and it blurs everything out except the external links like the one posted here. So these images aren't rendered the same way.

Video will also need a new blur callback. Strangely duplicating what is done for PostImage just doesn't work.

I don't know if these information are useful or not, but I hope so
